route/build/src/write_simoutput_pio.f90

```diff
@@ -299,9 +306,18 @@ SUBROUTINE defineFile(fname,           &  ! input: filename
  ! This is temporary
  if (routOpt==kinematicWave) then
   meta_rflx(ixRFLX%IRFroutedRunoff)%varFile = .false.
+  meta_rflx(ixRFLX%KWEroutedRunoff)%varFile = .false.
+ end if
+ if (routOpt==kinematicWaveEuler) then
+  meta_rflx(ixRFLX%IRFroutedRunoff)%varFile = .false.
+  meta_rflx(ixRFLX%KWTroutedRunoff)%varFile = .false.
  end if
  if (routOpt==impulseResponseFunc) then
   meta_rflx(ixRFLX%KWTroutedRunoff)%varFile = .false.
+  meta_rflx(ixRFLX%KWEroutedRunoff)%varFile = .false.
+ end if
+ if (routOpt==allRoutingMethods) then
+  meta_rflx(ixRFLX%KWEroutedRunoff)%varFile = .false.
```

Comment:
This is just temporary.   
For now, `allRoutingMethod` option forces the program to run Lagrangian KW and IRF excluding Euler KW.  
In the future, `allRoutingMethods` plans to be removed (suggestion).